networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -68,7 +68,7 @@ type Team @model {
 type TeamMember @model {
   id: ID!
   isFacilitator: Boolean
-  answers: [TeamAnswer] @hasMany(indexName: "byTeamMember", fields:["id"])
```

Comment:
I'm running into some issues with how `amplify` handles indexes with custom names. I'm reverting to the default naming (this generates `teamMemberAnswersId`) and will address this in the sorting/filtering PR.